spec/datetimeformat.html

```diff
@@ -58,6 +58,11 @@ <h1>Abstract Operations For DateTimeFormat Objects</h1>
           <td>*"second"*</td>
           <td>*"2-digit"*, *"numeric"*</td>
         </tr>
+        <tr>
+          <td>[[FractionalSecondDigits]]</td>
+          <td>*"fractionalSecondDigits"*</td>
+          <td>*"1"*, *"2"*, *"3"*</td>
```

Comment:
I didn't spot this before, but these should be number values, so the double-quotes need to be removed. 

This is required per [13.3.3 Internal slots](https://tc39.es/ecma402/#sec-intl.datetimeformat-internal-slots), which states that:
> The value of this field must be a list of records, each of which has a subset of the fields shown in Table 6, where each field must have one of the values specified for the field in Table 6.